index.js

```diff
@@ -39,6 +40,21 @@ module.exports = class TaskerTrailpack extends Trailpack {
     this.app.tasker = new Client(this.app, rabbit, taskerConfig.exchangeName)
     TaskerUtils.registerTasks(profile, this.app, rabbit)
 
+    return new Promise((resolve, reject) => {
+      const tasksPath = Path.join(__dirname, '../../api/tasks')
+      fs.stat(tasksPath, (err, fileStats) => {
```

Comment:
@ryanwilliamquinn The user (or a generator) needs to just add a line here: https://github.com/trailsjs/trails/blob/master/archetype/api/index.js. We should try not to `require` things during trails runtime, partially for this kind of reason.